src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,12 +11,12 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
-    public CommandResult(String feedbackToUser) {
+	public CommandResult(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
```

Comment:
Indentation looks to be a bit off! Are you using the eclipse auto-format. Just use that after making every change.